cabal.project

```diff
@@ -27,11 +27,14 @@ packages:
   bench/cardano-topology
   bench/locli
   bench/plutus-scripts-bench
-  bench/tx-generator
   trace-dispatcher
   trace-resources
   trace-forward
 
+if !(impl(ghc <9.6) || impl(ghc >=9.7))
+  packages:
```

Comment:
Does the performance team know about this?

Reply:
I assume they do. Its needed because the Plutus machinery is locked to a specific `X.Y` version of GHC.

Reply:
Please don't assume - just ask, or ping for a review.
Disabling `tx-generator` is wrong.
The recent change to the GHC constraint in `plutus-bench-scripts` was already too strict, and this has now propagateed.

Motivation for that constraint is:
* If we are on a GHC >= 9 we use the Plutus compiler plugin to build our scripts
* If we are on a GHC < 9 we fall back to pre-serialized script.

This does *NOT* block the `tx-generator`from being built.